h/stcbackground.h

```diff
@@ -0,0 +1,54 @@
+/*
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which 
+  accompanies this distribution, and is available at 
+  https://www.eclipse.org/legal/epl-v20.html
+
+  SPDX-License-Identifier: EPL-2.0
+
+  Copyright Contributors to the Zowe Project.
+*/
+
+#ifndef __STCBACKGROUND__
+#define __STCBACKGROUND__
+
+#include "stcbase.h"
+#define STC_BG_CB_LABEL_LEN 30
+
+struct STCIntervalCallbackData_tag;
+typedef int (*STCIntervalCallback)(void* server, 
```

Comment:
What's `server` for? I don't think we need it here. We already have `userData` for passing the data needed by the callbacks.

Reply:
removed server